File: Models/Account.cs
using System.Text.Json.Serialization;

namespace AccountManagerFunctionApp.Models
{
 public class Account
 {
 [JsonPropertyName("id")]
 public string? Id { get; set; }

 [JsonPropertyName("name")]
 public string? Name { get; set; }

 [JsonPropertyName("email")]
 public string? Email { get; set; }

 [JsonPropertyName("address")]
 public string? Address { get; set; }
 }
}

File: testjt/src/main/java/Testjt/models/Account.java
package Testjt.models;

import java.util.Map;
import java.util.HashMap;
import io.micronaut.core.annotation.Introspected;

@Introspected
public class Account {
    private Long id;
    private String name;
    private String email;
    private String createdAt;

    public Account() {}

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public Map<String, Object> toMap() {
        Map<String, Object> m = new HashMap<>();
        m.put("id", id);
        m.put("name", name);
        m.put("email", email);
        m.put("createdAt", createdAt);
        return m;
    }
}
